tgui/packages/tgui/interfaces/AntagInfoBlob.tsx

```diff
@@ -58,11 +58,13 @@ const Overview = (props) => {
   return (
     <Stack vertical>
       <Stack.Item bold fontSize="24px" textColor={BLOB_COLOR}>
-        You are the Blob!
+        Вы - Блоб!
       </Stack.Item>
-      <Stack.Item>As the overmind, you can control the blob.</Stack.Item>
       <Stack.Item>
-        Your blob reagent is:{' '}
+        В роли управляющего ядра вы можете управлять блобом.
```

Comment:
```suggestion
        В роли сверхразума вы можете управлять блобом.
```
Можно взять пример перевода с Starcraft. Сверхмозги сила

Reply:
> Можно взять пример перевода с Starcraft. Сверхмозги сила

А это точно будет звучать хорошо? Я и так мог, просто мне как то странно звучит

Reply:
Этакая отсылка будет к старику